src/main/resources/story/web_app/Trello End To End Scenarios.story

```diff
@@ -0,0 +1,90 @@
+Description: Test demoing VIVIDUS capabilities for Web Applications
+
+
+Lifecycle:
+Examples:
+{transformer=FROM_LANDSCAPE}
+|login-button           |By.xpath(//a[@class='Buttonsstyles__Button-sc-1jwidxo-0 kTwZBr'])                            |
+|email                  |sominasvet@gmail.com                                                                         |
+|email-field            |By.id(user)                                                                                  |
+|continue-button        |By.id(login)                                                                                 |
+|password-field         |By.id(password)                                                                              |
+|password               |P7@teS9aqVL                                                                                  |
+|login-submit-button    |By.id(login-submit)                                                                          |
+|create-new-board-button|By.xpath(//span[text()='Create new board'])                                                  |
+|board-title-field      |By.xpath(//input[@class='nch-textfield__input lsOhPsHuxEMYEb lsOhPsHuxEMYEb VkPAAkbpkKnPst'])|
+|create-button          |By.xpath(//button[@data-testid='create-board-submit-button'])                                |
+|list-name-input        |By.xpath(//input[@class='list-name-input'])                                                  |
+|list-add-button        |By.xpath(//input[@class='nch-button nch-button--primary mod-list-add-button js-save-edit'])  |
+|add-card               |By.xpath(//span[@class='js-add-a-card'])                                                     |
+|card-title-field       |By.xpath(//textarea[@class='list-card-composer-textarea js-card-title'])                     |
+|add-card-button        |By.xpath(//input[@value='Add card'])                                                         |
+|active-board           |By.xpath(//a[@aria-label='Board with tasks (currently active)'])                             |
+|invalid-password       |Lu9iomjlp0                                                                                   |
+
+
+Scenario: Verify login to Trello
+Given I am on page with URL `https://trello.com/`
```

Comment:
[improvement suggestion] Better to store main URL as a separate variable in properties file